kornia/morphology/gradient.py

```diff
@@ -0,0 +1,24 @@
+from kornia.morphology import Dilate, Erode
+import torch
+
+
+def gradient(tensor: torch.Tensor, kernel: torch.Tensor) -> torch.Tensor:
+    r"""
+        Returns the morphological gradient of an image,
+        (that means, dilation - erosion) applying the same kernel in each channel.
+        The kernel must have 2 dimensions, each one defined by an odd number.
+
+        Args
+           tensor (torch.Tensor): Image with shape :math:`(B, C, H, W)`.
+           kernel (torch.Tensor): Structuring element with shape :math:`(H, W)`.
+
+        Returns:
+           torch.Tensor: Dilated image with shape :math:`(B, C, H, W)`.
+
+        Example:
+            >>> tensor = torch.rand(1, 3, 5, 5)
+            >>> kernel = torch.ones(3, 3)
+            >>> gradient_img = gradient(tensor, kernel)
+        """
+
+    return Dilate(kernel)(tensor) - Erode(kernel)(tensor)
```

Comment:
isn't this similar to spatial_gradient ? /cc @ducha-aiki 
https://kornia.readthedocs.io/en/latest/filters.html?highlight=spatial%20gradient#kornia.filters.spatial_gradient

Reply:
@edgarriba  it is similar, but not the same and  it is better fit morphology module. So let it be here.